lib/get-contents.js

```diff
@@ -27,7 +27,7 @@ const getContents = ({ filepath, githubToken }) => {
 		const repo = repository.split('/')[1];
 
 		try {
-			const repoData = await octokit.repos.getContents({
+			const repoData = await octokit.repos.getContent({
```

Comment:
should it be `getContents` or `getContent`?

Reply:
never mind, found it referenced as
`octokit.repos.getContent({
  owner,
  repo,
  path,
});`
in the documentation https://octokit.github.io/rest.js/v18

Reply:
@emortong yeah that was a weird one because the documentation hasn't been updated 😱 but the release notes did mention it.

Reply:
![image](https://user-images.githubusercontent.com/102233/100856551-7b5d3c00-3483-11eb-899d-6417a3a8e45e.png)